packages/node_modules/@webex/plugin-meetings/src/locus-info/hostUtils.js

```diff
@@ -0,0 +1,36 @@
+import {} from '../constants';
+
+const hostUtils = {};
+
+hostUtils.parse = (host) => {
+  if (host) {
+    return {
+      id: hostUtils.getId(host)
+    };
+  }
+  return null;
+};
+
+hostUtils.getHosts = (oldHost, newHost) => {
+  const previous = oldHost && hostUtils.parse(oldHost);
+  const current = newHost && hostUtils.parse(newHost);
+  return {
+    previous,
+    current,
```

Comment:
What's the benefit in passing back previous/current if you have to pass them in to begin?

Reply:
Need both of them there to compare, but can remove the processing on the old one since it will have already been processed and the one that is previous is not already processed since locusInfo saves it raw and parsed